test/bench.hs

```diff
@@ -101,5 +115,11 @@ main = tinfoilBench [
                              ]
   , env ((,) <$> generate' arbitrary <*> generate' arbitrary) $ \ ~(sk, bs) ->
       bgroup "mac/hmacSHA256" $ [ bench "hmacSHA256" $ nf (hmacSHA256 sk) bs
-                             ]
+                                ]
+  , env genEd25519 $ \ ~(sk, pk, sig, msg) ->
+      bgroup "signing/ed25519" $ [
```

Comment:
Yeah, just a habit of mine with benchmarks. It has always compiled with 7.8 in the past.